lib/processors/manifestEnhancer.js

```diff
@@ -166,7 +204,11 @@ class ManifestEnhancer {
 				supportedLocales.push("");
 			} else if (fileNameWithoutExtension.startsWith(i18nBundlePrefix)) {
 				const locale = fileNameWithoutExtension.replace(i18nBundlePrefix, "");
-				supportedLocales.push(locale);
+				if (isValidPropertiesFilenameLocale(locale)) {
+					supportedLocales.push(locale);
+				} else {
+					log.verbose(`Skipping invalid locale '${locale}' for bundle '${i18nBundleUrl}'`);
```

Comment:
Still open: What should happen in this case? Maybe rather an warning or even error log?

Reply:
Decision: Log a warning and explain in commit message that the warning can be suppressed by manually maintaining the "supportedLocales".

Reply:
Warning added via https://github.com/SAP/ui5-builder/pull/1080/commits/11ab19f778cfe4a9cae6cd31efadc6b4e797ef6c

> To be checked: Built-in handling of private extensions at runtime (1Q, 2Q, 3Q).

This just turned out to be a confusion based on a wrong comment / commit message. It referred to `variants`, but actually it was supposed to be `extensions`.

Extensions are not considered by the runtime code at all (i.e. can't appear in requests).
Private use is checked for the special SAP supportability locales, but the resulting request does not include an extension or private use section but will rather be e.g. `en_US_saptrc` (where `saptrc` is a variant).